.github/workflows/action-image-scan-integration.yaml

```diff
@@ -0,0 +1,34 @@
+name: "docker-build"
+
+on:
+  push:
+    branches:
+      - master
+      - main
+  pull_request:
+  workflow_dispatch:
```

Comment:
What does this trigger catch?

Reply:
It is a manual trigger, but to work, needs to be on the default branch